### PR TITLE
Fix whitespace issues

### PR DESCRIPTION
**Pull request by pgaskin**
* Trailing whitespace.
* Spaces instead of tabs.
* Spaces between brackets and arguments.

### CODE REVIEW

**Review by pgaskin — COMMENTED** [review]
No change in functionality.

**Review by nefthy — APPROVED** [review]
changes are trivial

**Comment by pgaskin** [comment]
@flyingmutant

File: unidecomp.h [hunk 1]
```diff
@@ -11,7 +11,7 @@ static struct {
 	{   0xc3,   0x41 },	// Ã -> A,	 ̃ (303)
 	{   0xc4,   0x41 },	// Ä -> A,	 ̈ (308)
 	{   0xc5,   0x41 },	// Å -> A,	 ̊ (30a)
-	{   0xc6,   0x41 },	// Æ -> A,	
+	{   0xc6,   0x41 },	// Æ -> A,
```
Comment (flyingmutant) [inline unidecomp.h:14]:
```
/* This file is automatically generated. DO NOT EDIT!
Instead, edit gen_decomp.py and re-run. */
```

Comment (pgaskin) [reply unidecomp.h:14]: Fixed.

**Comment by flyingmutant** [comment]
Thanks!

**Pull request merged**